### PR TITLE
Adds QBS preview content on introduction page

### PR DESCRIPTION
**Pull request by yoganandkunche**
### What is the context of this PR?
Added preview content on the introduction page.

### How to review 
Verify that the preview content matches the spec.

https://docs.google.com/document/d/10v30YjBp9wX5wYJURSnPAZZ4BkbVLfCBcj414ri5OPQ/edit#

### CODE REVIEW

**Comment by charlotte-hirst** [comment]
@yoganandkunche it looks weird with questions and then one block of include/exclude. I know its done because all the questions have the same include/excludes. Maybe show @armstrongb  but not sure how much flexibility you have to change the structure?
quarterly business survey

**Comment by adoublebarrel** [comment]
Marking this as in progress while the content is being re-worked to no longer have the accordion.

File: data/en/2_0001.json [hunk 1]
```diff
@@ -6,25 +6,54 @@
...
-                    "type": "Basic",
-                    "title": "You will be asked to provide information for the business, including:",
+                },
+                "secondary_content": [{
+                    "title": "How we use your data",
                     "content": [{
                         "list": [
-                            "Employee numbers"
+                            "You cannot appeal your selection. Your business was selected to give us a comprehensive view of the UK economy.",
+                            "The information you provide contributes to the Labour Market Statistics."
```
Comment (jonnyshaw89) [inline data/en/2_0001.json:54]: Should `Labour Market Statistics` link to https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/bulletins/uklabourmarket/previousReleases

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: app/templates/partials/introduction/preview.html [hunk 2]
```diff
@@ -7,12 +7,19 @@ <h3 class="venus u-mt-s">{{ content.title }}</h3>
...
+    {% endfor %}
+</ul>
+{% endif %}
 {% endfor %}
 </div>
 <div class="accordion js-accordion">
         <div class="mars accordion__controls">
             <a class="accordion__control js-accordion-open-all u-hidden" data-ga="click" data-ga-category="Preview Survey" data-ga-action="Show all" tabindex="0" aria-hidden="true">Show all</a>
-            <a class="accordion__control js-accordion-close-all u-hidden" data-ga="click" data-ga-category="Preview Survey" data-ga-action="Hide all" tabindex="0" aria-hidden="true">Hide all</a>
+            <a class="accordion__control js-accordion-clogit ste-all u-hidden" data-ga="click" data-ga-category="Preview Survey" data-ga-action="Hide all" tabindex="0" aria-hidden="true">Hide all</a>
```
Comment (jonnyshaw89) [inline app/templates/partials/introduction/preview.html]: Is this change intentional?

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: data/en/2_0001.json (hunk 1 above)
Comment (yoganandkunche) [reply data/en/2_0001.json:54]: @jonnyshaw89 Checked with @charlotte-hirst before implementing this and was agreed that it will not be a link.

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by charlotte-hirst** [comment]
@yoganandkunche looks good

**Review by adoublebarrel — APPROVED** [review]
LGTM

**Comment by jonnyshaw89** [comment]
@yoganandkunche Squishy squishy
